### PR TITLE
Add option to provide Kubemanager env variables

### PR DESCRIPTION
**Pull request by Janek3d**

### CODE REVIEW

**Review by Dysproz — APPROVED** [review]

**Comment by Janek3d** [comment]
recheck

**Comment by Janek3d** [comment]
recheck ha

**Comment by Janek3d** [comment]
recheck

**Comment by psykulsk** [comment]
recheck

**Comment by Janek3d** [comment]
recheck

File: pkg/controller/kubemanager/kubemanager_controller.go [hunk 1]
```diff
@@ -220,6 +220,11 @@ func (r *ReconcileKubemanager) Reconcile(request reconcile.Request) (reconcile.R
 		return reconcile.Result{}, err
 	}
 
+	_, err = instance.CreateConfigMap(request.Name+"-"+instanceType+"-configmap-envs", r.Client, r.Scheme, request)
```
Comment (psykulsk) [inline pkg/controller/kubemanager/kubemanager_controller.go:223]: Nit: I would create a variable, for example `envVariablesConfigMap := request.Name+"-"+instanceType+"-configmap-envs"` and use it here and in line 436

**Review by psykulsk — APPROVED** [review]

**Pull request merged**